src/components/landing-page/create-production.tsx

```diff
@@ -97,6 +122,32 @@ export const CreateProduction = () => {
     }
   }, [createdProductionId, dispatch, reset]);
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const handleCopyProdUrlsToClipboard = (input: any) => {
+    if (input !== null) {
+      navigator.clipboard
+        .writeText(input)
+        .then(() => {
+          let timeout: number | null = null;
+          setCopiedUrl(true);
+          console.log("Text copied to clipboard");
+
+          timeout = window.setTimeout(() => {
+            setCopiedUrl(false);
+          }, 1500);
```

Comment:
Async actions must be handled inside of a useEffect hook, or they can't be cleaned up by react, causing unhandled errors in the console.

The way to achieve this here is to set a state directly, `setCopiedUrl(true)`, and have a `useEffect` hook set up that starts the timeout when this value toggles to `true`.